subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -153,6 +153,7 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 			lll = ftr->param;
 
 			lll_aux = lll->lll_aux;
+			lll->lll_aux = NULL;
```

Comment:
Active scanning and initiator will not work if ULL clears the aux context. `lll->lll_aux` is to differentiate LLL scheduled active scanning/initiator. Check using `tests/bluetooth/bsim/bsim_test_advx`  and with/without `CONFIG_BT_CTLR_ADV_AUX_OFFSET_CONSTANT=y`